html/Teleport_in_out.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+    <head>
+        <meta charset="UTF-8">
+        <title>teleport</title>
+        <link rel="stylesheet" href="Teleport_in_out.css" />
+    </head>
+
+    <body>
+        <h1 class ="header">Rule</h1>
+
+        <div class="text">
+            <div class="pic1">
+                <img width='200px;' src='https://st4.depositphotos.com/1001599/24639/v/600/depositphotos_246399964-stock-illustration-human-teleportation-hand-drawn-outline.jpgg' />
+            </div>
+    
+           
+                <div class="t11">
+                <p class="p_with_color">Teleport in cell</p>
+                <p>USE: infinite number of times</p> 
+                <p>COST: free</p>                
+            </div>
+    
+            <div class="t12">
+                <p>Teleport in cell helps hero to pass more then 1 cell in 1 step</p>
+            </div>
+           
+        </div>
+        
+            <div class="text">
```

Comment:
you can unite  <div class="pic1"> and <div class="t11"> with a div, u can use in css for web-elements display (exampe:  " display: flex; " ). and etc.